docs/book/src/developer/architecture/controllers/machine.md

```diff
@@ -62,14 +73,19 @@ status:
 
 ### Infrastructure provider
 
-The InfrastructureMachine object **must** have a `status` object.
+The InfrastructureMachine object **must** have both `spec` and `status` objects.
+
+#### Required `spec` fields
+
+The `spec` object **must** at least one field defined:
+
+* `providerID` - a cloud provider ID identifying the machine. This is often set by the cloud-provider-controller.
```

Comment:
What do you mean by `cloud-provider-controller` in this context?

Reply:
typo, infra provider controller

Reply:
I'd just say `infrastructure providers`, like we do elsewhere

Reply:
actually just deleting this, it's clarified above.